dGame/dInventory/Item.h

```diff
@@ -193,9 +193,8 @@ class Item final
 
 	/**
 	 * Uses this item if its non equip, essentially an interface for the linked GM
-	 * @return whether the use was successful, e.g. the skill was cast
 	 */
-	bool UseNonEquip();
+	void UseNonEquip();
```

Comment:
Is there a reason why this functionality was removed?

Reply:
It is unused.  The return value was always discarded and is deprecated now

Reply:
Ah understood, I was unaware that it wasn't used.